CODEOWNERS

```diff
@@ -12,4 +12,4 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-* @pjanotti @sjkaris @rghetia @tedsuo @jmacd @iredelmeier @paivagustavo @krnowak @lizthegrey
+* @pjanotti @sjkaris @rghetia @tedsuo @jmacd @iredelmeier @paivagustavo @krnowak @lizthegrey @iredelmeier
```

Comment:
see here, you're already in the file. but we need to update @open-telemetry/go-approvers to reflect the new additions.

Reply:
😂😂😂 totally missed that!

I sent a request to join the team.